ansible/adhoc/lock_unlock_instances.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- hosts: "{{ target_hosts | default('all') }}"
```

Comment:
I don't think target_hosts is necessary, this can just be `hosts: cluster`. If you want to only run on some _as an adhoc_ you can just use `ansible-playbook --limit ...` which is what we do for e.g. the rebuild adhoc.

I know you suggested being able to "tweak" this for rebuild groups (presumably when running this from site, rather than as an ad-hoc, but TBH with the way that's passed via `vars:` at the moment you can't override it from inventory anyway, so I wouldn't bother.